test-configs.yaml

```diff
@@ -1652,6 +1669,11 @@ test_configs:
       - baseline
       - kselftest
 
+  - device_type: imx6sll-evk
+    test_plans:
+      - baseline
+      - kselftest
```

Comment:
Please drop the kselftest from these new test-plans.  We're reworking kselftest, and going forward, kselftest will require boards to support NFSroot.